powershell-adapter/Tests/TestAdapter/testadapter.resource.ps1

```diff
@@ -48,16 +48,19 @@ switch ($Operation) {
                 description    = 'TestCase resource'
             } | ConvertTo-Json -Compress
     }
-    { @('Get','Set','Test','Export') -contains $_ } {
+    { @('Get','Set','Test') -contains $_ } {
+        "Operation: $Operation" | Write-DscTrace
 
-        # TestCase 1 = 'Verify adapted_dsc_type field'
-        if (($inputobj.TestCaseId -eq 1 ) -or ($_ -eq 'Export')){
-            $result = $inputobj.adapted_dsc_type -eq 'Test/TestCase'
-            $result = @{'TestCaseId'=1; 'Input'=''; result = $result } | ConvertTo-Json -Depth 10 -Compress
-            return $result
+        if (($inputobj.resources.properties.TestCaseId -eq 1 ) -or ($_ -eq 'Export')){
```

Comment:
```suggestion
        if ($inputobj.resources.properties.TestCaseId -eq 1) {
```
Is the Export check still needed if that test case is handled below?

Reply:
Good catch, that check is no longer needed